### ISSUE TITLE
Hide scheme-container if there are no Authorization/schemes/servers

### ISSUE THREAD

**Issue by hkosova** [swagger-api/swagger-ui#5029]
Swagger UI version: 3.19.5

If an API definition contains no securities and no `schemes` (OAS2) or `servers` (OAS3), Swagger UI still renders an empty `scheme-container`. It serves no purpose and just takes up vertical space (see the screenshot below).

To use vertical space more efficiently, consider not displaying `scheme-container` if it does not have any contents.

### Content & configuration

Swagger/OpenAPI definition:

2.0:
 ```yaml
swagger: "2.0"
info:
  title: test
  version: 1.0.0
  description: Lorem ipsum
host: "petstore.swagger.io"
basePath: "/v2"
paths:
  /foo:
    get:
      responses:
        200:
          description: OK
 ```

3.0:
```yaml
openapi: 3.0.0
info:
  title: test
  version: 0.0.0
  description: Lorem ipsum
paths:
  /foo:
    get:
      responses:
        '200':
          description: OK
```

### Actual result
Huge empty vertical space between the API description and operations.

![schemes-container](https://user-images.githubusercontent.com/8576823/48497065-74cfdd80-e844-11e8-84c4-894bca83959c.png)

### Expected result

`scheme-container` is not displayed if it does not have any contents.

![image](https://user-images.githubusercontent.com/8576823/48497423-22db8780-e845-11e8-94b6-53755b8b1524.png)

**Comment by hkosova** [comment]
Looks like PR https://github.com/swagger-api/swagger-ui/pull/4950 is supposed to fix this.

**Comment by shockey** [comment]
Solved in #4950. 

Thanks for filing, @hkosova!
